docs/usage/shoot_operations.md

```diff
@@ -40,3 +40,17 @@ Please note that only the token (and basic auth password, if enabled) are exchan
 ```bash
 kubectl -n garden-<project-name> annotate shoot <shoot-name> gardener.cloud/operation=rotate-kubeconfig-credentials
 ```
+
+## Restart systemd services on particular worker nodes
+
+It is possible to make Gardener restart particular systemd services on your shoot worker nodes if needed.
+The annotation is not set on the `Shoot` resource but directly on the `Node` object you want to target.
+For example, the following will restart the `kubelet`:
+
+```bash
+kubectl annotate node <node-name> worker.gardener.cloud/restart-systemd-services=kubelet
```

Comment:
I believe it would be better if the documentation shows that multiple services can be restarted with single annotation, e.g.:
```suggestion
For example, the following will restart the `docker` and `kubelet`:

```bash
kubectl annotate node <node-name> worker.gardener.cloud/restart-systemd-services=docker,kubelet
```